### PR TITLE
Add raw message for Ansible

### PR DESCRIPTION
**Pull request by ofedoren**
Adds a link which will open a modal with raw log message:
ScreenShot-1634821236048
ScreenShot-1634821251284

### CODE REVIEW

**Comment by ares** [comment]
I think that's a great functionality. Could the Raw be actually a button in a separate column? And perhaps change the label to "Raw data" so it's clearer?

**Comment by ofedoren** [comment]
Thanks for the idea, @ares! Due to suggested changes I've updated the tables to have PF4 style. Unfortunatelly due to PF4 button style (even the small one) there is a lot of whitespaces on Ansible page :/
Ansible:
ScreenShot-1635519704828
Puppet:
ScreenShot-1635519717292

**Comment by lzap** [comment]
@Ron-Lavi can you please test and review? We will be replacing this with new component(s) so perhaps just a quick look and test. Oh I asked Mel to give you push permissions here! :)

**Comment by lzap** [comment]
@Ron-Lavi FYI the idea is to show full log as Ansible sends it, this is becuase Ansible does not have constent report lines like Puppet do. we do construct a string called "friendly_message" that sort of captures the most used Ansible modules in a human-readable form.

File: webpack/src/Router/HostReports/ShowPage/Components/ReportLogs/Components/EmptyLogsRow.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        'No results match this filter criteria. Clear all filters and try again.'
+      )}
+    </EmptyStateBody>
+    <EmptyStateSecondaryActions>
+      <Button variant="link" onClick={onClear}>
+        {__('Clear all filters')}
+      </Button>
+    </EmptyStateSecondaryActions>
+  </EmptyState>
+);
```
Comment (Ron-Lavi) [inline webpack/src/Router/HostReports/ShowPage/Components/ReportLogs/Components/EmptyLogsRow.js:21]: can you use the [PF4 empty state](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js) from core?

**Review by Ron-Lavi — COMMENTED** [review]
code wise looks good to me,
I really like the composable table implementations 👍🏼
didn't test it out yet.

**Comment by ofedoren** [comment]
Thanks, @Ron-Lavi, updated to use the component from core.

**Comment by adamruzicka** [comment]
@Ron-Lavi Anything to add here?

**Comment by Ron-Lavi** [comment]
Thanks @ofedoren !
code-wise looks good to me 👍🏼

**Review by Ron-Lavi — APPROVED** [review]

**Comment by lzap** [comment]
@Ron-Lavi if you dont have merge permissions let me know, you should have it.

**Pull request merged**